### PR TITLE
Gentle queue abort with pcntl_signal handler

### PR DESCRIPTION
**Pull request by m0rth1um**
Catch process signals with pcntl_signal and finish current job before stopping loop

### CODE REVIEW

File: src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php [hunk 1]
```diff
@@ -2,11 +2,28 @@
 
 namespace josegonzalez\Queuesadilla\Worker;
 
+declare(ticks=1);
```
Comment (josegonzalez) [inline src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php]: Do we actually need this? iirc `ticks` are only necessary for versions under 5.3, which this project cannot support.

**Comment by josegonzalez** [comment]
Nice!

/cc @cleptric since you use this in prod :)

**Comment by cleptric** [comment]
@josegonzalez @m0rth1um is a co-worker of mine 🙂

**Comment by josegonzalez** [comment]
Tell your coworker that their tests failed :D

**Comment by josegonzalez** [comment]
Tell your coworker that their tests failed :D

File: src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php (hunk 1 above)
Comment (m0rth1um) [inline src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php]: Yes. It's necessary. When this line is not there, <kbd>Ctrl</kbd>+<kbd>C</kbd> has no effect.

**Comment by m0rth1um** [comment]
@josegonzalez I fixed the tests.

**Pull request merged**